nextflow_schema.json

```diff
@@ -226,12 +226,6 @@
                     "description": "Show all params when using `--help`",
                     "hidden": true,
                     "help_text": "By default, parameters set as _hidden_ in the schema are not shown on the command line when a user runs with `--help`. Specifying this option will tell the pipeline to show all parameters."
-                },
-                "enable_conda": {
```

Comment:
is this the change nf-core did by no longer using this functionality?

Reply:
this is one of them yes